genie-biogem/src/fortran/biogem_data.f90

```diff
@@ -971,54 +971,93 @@ SUBROUTINE sub_data_update_tracerrelationships()
     ! -------------------------------------------------------- !
     ! UPDATE REDFIELD RELATIONSHIPS
```

Comment:
Would say
UPDATE REDFIELD RELATIONSHIPS DURING REMINERALISATION

Reply:
[will improve commenting in a future merge]